packages/cli/src/lib/utils/process.ts

```diff
@@ -34,3 +35,31 @@ export function spawnProcess(
     });
   });
 }
+
+export async function spawnProcessStdio(
+  command: string,
+  args: string[],
+  options: SpawnOptions = {}
+): Promise<{stdout: string; stderr: string}> {
+  return new Promise((resolve) => {
+    const stdio = {stdout: '', stderr: ''};
```

Comment:
Add exitCode to the stdio (prolly should be renamed to something else like `processOutputs`)
```suggestion
    const stdio = {stdout: '', stderr: '', exitCode: null};
```